impl/src/bitfield.rs

```diff
@@ -254,20 +248,25 @@ impl BitfieldStruct {
     fn expand_getters_and_setters(&self) -> Result<TokenStream2> {
         let mut expanded = quote! {};
         let mut offset = Punctuated::<syn::Expr, Token![+]>::new();
-        offset.push(syn::parse_quote!{ 0 });
+        offset.push(syn::parse_quote! { 0 });
         for (n, field) in self.ast.fields.iter().enumerate() {
             use crate::ident_ext::IdentExt as _;
-            let field_name = field.ident
+            let field_name = field
+                .ident
                 .as_ref()
                 .map(ToString::to_string)
                 .unwrap_or(format!("{}", n));
             let getter_name = syn::Ident::from_str(format!("get_{}", field_name));
             let setter_name = syn::Ident::from_str(format!("set_{}", field_name));
```

Comment:
Should make use of `format_ident`: https://docs.rs/quote/1.0.7/quote/macro.format_ident.html

Reply:
good catch

Reply:
This had led me to add a test case for [raw identifiers](https://doc.rust-lang.org/stable/rust-by-example/compatibility/raw_identifiers.html) as field names, which are thus supported but failed previously.